File: hollow_inverted_pyramid.py
rows=int(input("Enter the number of rows: "))

for i in range (rows,0,-1):
    for j in range (rows,0,-1):
        if j>i:
            print(" ",end='')
    for k in range (1,2*i):
        if i!=rows:
            if k==1 or k==2*i-1:
                print("*",end='')
            else:
                print(" ",end='')
        else:
            print("*",end='')
    print('\n')

File: hollow_inverted_pyramid.c
#include<stdio.h>

int main(){
    int rows;
    printf("Enter the number of rows: ");
    scanf("%i",&rows);

    for (int i=1; i<=rows; i++)
    {
        for (int j=1; j<i; j++)
        {
            printf(" ");
        }
        for (int k=2*rows-1; k>=2*i-1; k--)
        {
            if (i!=1)
            {
                if (k==2*rows-1 || k==2*i-1)
                {
                    printf("*");
                }
                else
                {
                    printf(" ");
                }
            }
            else
            {
                printf("*");
            }
        }
        printf("\n");
    }
    
    return 0;
}